File: npm/src/utils/keywords.ts
export default class Keywords {
    public static importKey: string = "$import \""; 
    public static singleLineComment: string = "//";
    public static multiLineCommentStart: string = "/*";
    public static multiLineCommentEnd: string = "*/";

    public static relativeJPath: string = "$.";
    public static relativeJPathRegex: RegExp = new RegExp("[$][.][.A-Za-z0-9]*", "g");
    
    public static runTimeValsRegex: RegExp = new RegExp("[<][A-Za-z0-9]*[>]", "g");

    public static pathRegex: RegExp = new RegExp("[A-Za-z0-9:\\.]+", "g");

    public static removeWithSucComa = (key: string, value: string): RegExp => new RegExp("\"" + key.split(".")[key.split(".").length - 1] + "\":\\s*\"*" + value + "\"*,*\\s*\\t*\\r*\"", "g");
    public static removeWithPreComa = (key: string, value: string): RegExp => new RegExp(",*\n*\s*\\t*\r*\"" + key.split(".")[key.split(".").length - 1] + "\":\\s*\"*" + value + "\"*", "g");
}

File: python/cjson/src/utils/keywords.py
class Keywords:
    import_key: str = "$import \""
    single_line_comment: str = "//"
    multiLine_comment_start: str = "/*"
    multiLine_comment_end: str = "*/"
    relative_jpath: str = "$."
    relative_jpath_regex: str = "[$][.][.A-Za-z0-9]*"
    runtime_vals_regex: str = "[<][A-Za-z0-9]*[>]"

    def remove_with_suc_coma(key: str, value: str) -> str:
        return "\n*\s*\\t*\r*\"" + key.split(".")[-1] + "\":\\s*\"*" + value + "\"*,*\\s*\\t*\\r*"

    def remove_with_pre_coma(key: str, value: str) -> str:
        return ",*\n*\s*\\t*\r*\"" + key.split(".")[-1] + "\":\\s*\"*" + value + "\"*"